packages/core/helper-plugin/src/components/FilterListURLQuery.tsx

```diff
@@ -162,8 +186,17 @@ const AttributeTag = ({ attribute, filter, onClick, operator, value }: Attribute
   return (
     <Box padding={1}>
       <Tag onClick={handleClick} icon={<Cross />}>
-        {content}
+        <span onClick={handleContentClick}>{content}</span>
```

Comment:
please, span is a text, its behaviour is not to have an onClick, if you want to handle the onClick in different way you can add a condition in the handleClick handler, there is not need of two handlers for the same thing, you can differentiate the behaviour base on the event target element that triggered the handleClick for example